packages/@okta/vuepress-site/.vuepress/nav/main.js

```diff
@@ -29,25 +41,18 @@ module.exports = [
           { title: 'Revoking Tokens', link: '/authentication-guide/tokens/revoking-tokens/'}
         ]
       },
-      { title: 'Generic OpenID Connect IdP', link: '/authentication-guide/generic-oidc/'}
-    ]
-  },
-  {
-    title: 'Use Cases',
-    links: [
-      { title: 'Inline Hooks', link: '/use_cases/inline_hooks/'},
-      { title: 'Event Hooks', link: '/use_cases/event_hooks/'},
-      { title: 'Authentication', link: '/use_cases/authentication/'},
-      { title: 'Events API Migration', link: '/use_cases/events-api-migration/'},
-      { title: 'Multi-Factor Authentication', link: '/use_cases/mfa/'},
-      { title: 'API Access Management', link: '/use_cases/api_access_management/'},
-      { title: 'Relationships with Linked Objects', link: '/use_cases/relationships/'}
+      { title: 'Generic OpenID Connect IdP', link: '/authentication-guide/generic-oidc/'},
+      { title: 'Add User Consent to Your Authentication Flow', link: '/docs/how-to/add-user-consent-to-flow/'},
+      { title: 'Upload Your Own Certificates for Outbound SAML Apps', link: '/docs/how-to/byo_saml/'},
+      { title: 'Create an ID Token or Access Token Containing a Groups Claim', link: '/docs/how-to/creating-token-with-groups-claim/'},
+      { title: 'Share Application Key Credentials for IdPs Across Apps', link: '/docs/how-to/sharing-cert/'},
+      { title: 'Upgrade SAML Apps to SHA256', link: '/docs/how-to/updating_saml_cert/'}
     ]
   },
   {
-    title: 'API Reference',
+    title: "Reference",
```

Comment:
Note that the main nav "Docs" still has "API Reference"

Reply:
Good catch. Indeed, the global nav dropdown and Reference section index should also be updated to "Reference". But, no opinion from me about whether that should happen in this PR or separately.

<img width="697" alt="reference" src="https://user-images.githubusercontent.com/26014482/57420580-e0678780-71a3-11e9-8bec-f67de0050c19.png">